src/mame/drivers/hornet.cpp

```diff
@@ -257,6 +287,11 @@ f
       056800  - Konami Custom (QFP80)
 
 
+    GQ830 PWB(J) I/O PCB
+    ------------------------------------------
+    Note: This is simply another ADC12138. It uses two 5k potentiometers for X/Y analog inputs sending the data back
+    to the main pcb (either to the main ADC12138 or PPC403GA's serial port). Silent Scope EX would later use this pcb.
```

Comment:
The “either to the main ADC12138 or PPC403GA's serial port” phrase is unfounded speculation.  If you can’t show how the communication works, don’t speculate like this.  It’s unhelpful.